packages/round-manager/src/features/round/ViewRoundResults/ViewRoundResults.tsx

```diff
@@ -36,22 +37,49 @@ const distributionOptions = [
 export default function ViewRoundResults() {
   const { id } = useParams();
   const roundId = utils.getAddress(id?.toLowerCase() ?? "");
-  const { data: matches } = useRoundMatchingFunds(roundId);
+  // eslint-disable-next-line prefer-const
+  let { data: matches, isLoading } = useRoundMatchingFunds(roundId);
   const debugModeEnabled = useDebugMode();
+  const { data: round } = useRound(roundId);
+  const { uploadedData, uploadedFilename, onDrop, error } = useFileUpload();
 
   const [distributionOption, setDistributionOption] = useState("keep");
+  const [currentRoundSaturation, setCurrentRoundSaturation] = useState(0);
+  const [isSaturated, setIsSaturated] = useState(true);
+  const [revisedMatches, setRevisedMatches] = useState<any>(null);
+
+  useEffect(() => {
+    if (distributionOption === "scale" && !isSaturated && matches) {
+      const scaleFactor = 1 / currentRoundSaturation;
+      const newMatches = matches.map((match) => {
+        return {
+          ...match,
+          revisedMatched: match.matched * scaleFactor,
+        };
+      });
+      setRevisedMatches(newMatches);
+    } else {
+      setRevisedMatches(null);
+    }
+  }, [currentRoundSaturation, distributionOption, isSaturated, matches]); 
 
-  const onDrop = useCallback((acceptedFiles: File[]) => {
-    acceptedFiles.forEach((file: File) => {
-      const reader = new FileReader();
-      reader.onload = () => {
-        if (reader.result) {
-          /**/
-        }
-      };
-      reader.readAsText(file);
-    });
-  }, []);
+  useEffect(() => {
+    if (matches && round) {
+      console.log(round)
+      console.log(matches)
+      const matchTotal = matches.reduce(
```

Comment:
Here is a point of confusion: 

when we sum the match total (+= each project match amount) then this should not necessarily add up to the match pool amount - but in every case it seems to. Which will then never enable the radio buttons for recalculations. 

Is this correct?

Reply:
do you have a match URL to test the results please?